src/main/resources/application.properties

```diff
@@ -1,31 +1,106 @@
 # Properties file
 app.propDir=file:${catalina.base}/webapps
-spring.config.import=optional:classpath:env/${fr.insee.pogues.env:dev}/pogues-bo.properties,optional:${app.propDir}/pogues-bo.properties,optional:${app.propDir}/rmes-pogbo.properties
+
+
+# SSL: old properties, add to be false now
+fr.insee.pogues.force.ssl=false
+
+# Swagger
+fr.insee.pogues.api.host=localhost:8080
+fr.insee.pogues.api.name=
+fr.insee.pogues.api.scheme=http
+
+# CORS filter: empty means no CORS allowed, use * to allow all origins (not recommended)
+fr.insee.pogues.cors.allowedOrigin=
 
 # Database
+fr.insee.pogues.persistence.database.host=
+fr.insee.pogues.persistence.database.port=
+fr.insee.pogues.persistence.database.schema=
+fr.insee.pogues.persistence.database.user=
+fr.insee.pogues.persistence.database.password=
+fr.insee.pogues.persistence.database.table=pogues
+fr.insee.pogues.persistence.database.driver=org.postgresql.Driver
+
 spring.datasource.url=jdbc:postgresql://${fr.insee.pogues.persistence.database.host}:${fr.insee.pogues.persistence.database.port}/${fr.insee.pogues.persistence.database.schema}
 spring.datasource.username=${fr.insee.pogues.persistence.database.user}
 spring.datasource.password=${fr.insee.pogues.persistence.database.password}
 # Max pool size is 8 normally but to leave room to DDI-Access-services we put 4
 spring.datasource.hikari.minimum-idle=4
 spring.datasource.hikari.maximum-pool-size=4
 
-# SSL
-fr.insee.pogues.force.ssl=false
+# Protected stamp: only members associated with this stamp can edit questionnaires that belong to it
+fr.insee.pogues.stamp.restricted=DG75-L120
```

Comment:
We might review this stamp feature, and have a non-Insee-related default value here